packages/blade/src/_helpers/types.ts

```diff
@@ -11,4 +11,11 @@ type DotNotationColorStringToken<TokenType> = {
     : DotNotationColorStringToken<TokenType[K]>}`;
 }[keyof TokenType];
 
-export { DotNotationColorStringToken };
+type Required<T> = Exclude<T, undefined | null>;
+
+type ValueOf<
```

Comment:
what is the significane of this? and what does this do?

Reply:
This will extract the values of all the keys inside an object/array

```
{
	"a": 1,
	"b": 2,
	"c": 3,
}
```